packages/@react-types/actionbar/src/index.d.ts

```diff
@@ -14,17 +14,20 @@ import {CollectionBase, DOMProps, StyleProps} from '@react-types/shared';
 import {Key, ReactNode} from 'react';
 
 export interface ActionBarProps<T> extends CollectionBase<T> {
```

Comment:
I didn't like the definition of `items` in the props table, but it is generic coming from the CollecitonBase. The `items` are for the ActionGroup so the definition seems wrong, but I don't think we can fix it.